engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -336,7 +336,7 @@ class CromwellApiServiceSpec extends AsyncFlatSpec with ScalatestRouteTest with
         check {
           status should be(StatusCodes.OK)
           val decoder: Decoder = Gzip
-          val result = Await.result(Unmarshal(decoder.decode(response)).to[JsObject], 1.second)
+          val result = Await.result(Unmarshal(decoder.decodeMessage(response)).to[JsObject], 1.second)
```

Comment:
I *think* this fits the pattern that lets you just say `val result = responseAs[JsObject]`, avoiding the whole convoluted `Await.result` and decoding boilerplate :crossed_fingers: 

(and the examples below too, if it works here)